src/dialect/dialect-adapter.ts

```diff
@@ -9,6 +9,14 @@ import { Kysely } from '../kysely.js'
  * it. For that there's a `supportsTransactionalDdl` boolean in this interface.
  */
 export interface DialectAdapter {
+  /**
+   * Whether or not this dialect supports `if not exists` in creation of tables/schemas/views/etc.
+   *
+   * If this is false, tables and schemas are created without `if not exists` in
+   * migrations. This is not a problem if the dialect supports transactional DDL.
```

Comment:
note to self, this should say that Kysely's internal migration tables and schemas.